packages/aragon-wrapper/src/index.js

```diff
@@ -1319,6 +1320,42 @@ export default class Aragon {
     return []
   }
 
+  /**
+   * Calculate the transaction path for a transaction to `destination`
+   * that invokes `methodName` with `params`.
+   *
+   * @param  {string} destination An address of an uninstalled smart contract application
+   * @param  {string} method ABI of smart contract method
+   * @param  {Array<*>} params
+   * @return {Promise<Array<Object>>} An array of a single Ethereum direct transaction
+   */
+  async getUninstalledAppTransactionPath (destination, method, params) {
```

Comment:
Instead of having two functions, we could probably just combine this with `getTransactionPath()` and handle it as a case where the `destination` is not an installed app.

Reply:
Hey @sohkai - originally I tried to approach external tx pathing this way, but what's the best strategy to handle the ABI of the external method without breaking things? 

One approach is to include an optional methodAbi param:

```js
 async getTransactionPath (destination, methodName, params, finalForwarder, methodAbi = []) { ... }
```

Another way that feels prettier, but also like a much bigger change, is to adjust the `getTransactionPath` method to take `method` (includes the abi) rather than just the `methodName`, and then grab the name from the `method` object. This is pretty much exactly what `getUninstalledAppTransactionPath` does: [method is used as abi](https://github.com/aragon/aragon.js/pull/328/files#diff-5b914b5b526a57f89974fa2fe009ee80R1338] & [using method.name as methodName](https://github.com/aragon/aragon.js/pull/328/files#diff-5b914b5b526a57f89974fa2fe009ee80R1344)

Although more concise, this strategy _feels_ like its changes could propagate to other parts of aragon.js and potentially break things. 

Am i missing something? or is there another strategy you like better that I didn't think of?